test/test_distributions_util.py

```diff
@@ -170,6 +170,42 @@ def test_standard_gamma_batch():
         assert_allclose(samples[i], standard_gamma(rngs[i], alphas[i]))
 
 
+@pytest.mark.parametrize('prim', [
+    xlogy,
+    xlog1py,
+])
+def test_binop_batch_rule(prim):
+    bx = np.array([1., 2., 3.])
+    by = np.array([2., 3., 4.])
+    x = np.array(1.)
```

Comment:
What if this was just a python numeric type - would the test work fine?

Reply:
Yup, it works fine for `x = 1, y=2`. To not worry about python scalar, we just need to use `np.shape(x)` or `np.reshape(x, ...)` instead of `x.shape`, `x.reshape(...)`. :D